packages/cli/src/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Command } from 'commander';
+import { createPieceCommand } from './lib/commands/create-piece';
+
+const program = new Command();
+
+program.version('0.0.1').description('Activepieces CLI');
+
+program
+  .command('create-piece')
+  .argument('<pieceName>', 'piece name')
+  .argument('<packageName>', 'package name')
+  .requiredOption(
```

Comment:
the type can be interactive cli that allow the user to choose between custom and community

Reply:
I believe the packageName isn't necessary since it can be deduced from the name. For instance, if the user inputs a piece name like "hello," the default package name would be @activepieces/piece-hello unless the user specifies otherwise.